physical-and-link-layers/Readme.md

```diff
@@ -0,0 +1,31 @@
+### Goal:
+1. Fix problems with links on all hosts.
+2. Configure network interfaces on all hosts and management on switches.
+3. Outline in blue all broadcast domains, and in red all collision domains.
+
+### Solution:
+1. For each PC need to set up the IP address in FastEthernet0 > IP Configuration and check if the port is turned on.
+2. Set up sw1 with the following commands:
+* =>`en`
+* =>`conf t`
+* =>`interface vlan1`
+* =>`ip add 192.168.1.1 255.255.0.0`
+* =>`no shut`
+3. Set up sw2 with the following commands:
+* =>`en`
+* =>`conf t`
+* =>`interface vlan1`
+* =>`ip add 192.168.1.2 255.255.0.0`
+* =>`no shut`
+4. It’s necessary to set up bandwidth 100 Mbps and full-duplex for each PC(except PC-PT 10.0.0.8 and PC-PT 10.0.0.9 because they are connected via Hub)
```

Comment:
Why is it necessary? What's wrong with auto-discovery?

Reply:
Auto-discovery works correctly only in case if PC and Switch are connected between each other via Streight-Through cable if connection established via Cross-Over, then Auto mode doesn't work properly.

Reply:
Bullshit. It works fine in any cases. I never saw anything like that. It just works.
Maybe in early 2000 that was the case, when the autodiscovery feature was still something new.

Reply:
Well, that is old technology, as I mentioned.